django/db/backends/base/schema.py

```diff
@@ -897,14 +896,22 @@ def _model_indexes_sql(self, model):
             return []
         output = []
         for field in model._meta.local_fields:
-            if self._field_should_be_indexed(model, field):
-                output.append(self._create_index_sql(model, [field], suffix=""))
+            output.extend(self._field_indexes_sql(model, field))
 
         for field_names in model._meta.index_together:
             fields = [model._meta.get_field(field) for field in field_names]
             output.append(self._create_index_sql(model, fields, suffix="_idx"))
         return output
 
+    def _field_indexes_sql(self, model, field):
+        """
+        Return all index SQL statements for the specified field, as a list.
```

Comment:
I see you just copied the docstring above but it seems like "Return a list of all..." would be simpler.